lib/matplotlib/axes/_axes.py

```diff
@@ -3317,6 +3317,10 @@ def get_next_color():
                 else:
                     raise TypeError(
                         'autopct must be callable or a format string')
+                
+                if mpl.rcParams["text.usetex"] or 'usetex' in textprops:
```

Comment:
`textprops` overrides the RC param, and its value matters

Reply:
Resolved, Thanks!